BackupPC_report

```diff
@@ -23,10 +23,12 @@ my $bpcUser = 'backuppc';
 delete @ENV{qw(IFS CDPATH ENV BASH_ENV)};
 $ENV{PATH} = '/usr/bin';
 
-our ( $opt_a, $opt_s, $opt_w, $opt_help, $opt_man );
-GetOptions( "a", "s", "w=f", "help|?", "man" ) || pod2usage(2);
+our ( $opt_a, $opt_f, $opt_s, $opt_w, $opt_help, $opt_man );
+$opt_f = "report";
+GetOptions( "a", "f=s" => "report", "s", "w=f", "help|?", "man" ) || pod2usage(2);
 pod2usage( -exitstatus => 0, -verbose => 2 ) if $opt_man;
 pod2usage(1) if $opt_help;
+pod2usage( -msg => "Unsupported value for -f", -exitstatus => 2 ) if $opt_f !~ /(report|csv)/i;
```

Comment:
Are you using non-strict checks on purpose? I mean something like `blacsvbla` will match. I'd expect `eq/ne` or anchored regexps .

Reply:
should be hardened and checks for "allowed" values for `$opt_f`